uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Mail/Message.php

```diff
@@ -182,4 +189,14 @@ private function getLeaveApprovers() {
   private function getNotificationReceivers() {
     return NotificationReceiver::getReceiversIDsForAbsenceType($this->leaveRequest->type_id);
   }
+
+  /**
+   * Checks whether the template is a valid template and extends the
```

Comment:
**Checks whether the template is a valid template or not ** as a description for the method should be enough .. no need to describe the implementation details that could change.